intrinsic_sdk_cmake/cmake/find_replace_fix_imports.cmake

```diff
@@ -16,10 +16,10 @@ set(FIND_LIST
   "from google.type"
 )
 set(REPLACE_LIST
-  "from googleapis.google.api"
-  "from googleapis.google.longrunning"
-  "from googleapis.google.rpc"
-  "from googleapis.google.type"
+  "from googleapis.api"
+  "from googleapis.longrunning"
+  "from googleapis.rpc"
+  "from googleapis.type"
```

Comment:
I'm surprised by this change, the intention was to namespace it, not rename it. Maybe the logic that is namespacing the files into a separate directory is broken instead. (though I thought my testing would have seen this too)

Reply:
I need to revert the change and spin a container to actually test this but IIRC there was a missing `google` subfolder in the Python package